src/graphnet/models/cnn/theos_muonE_upgoing.py

```diff
@@ -0,0 +1,411 @@
+"""CNN used for muon energy reconstruction in IceCube.
+
```

Comment:
`src/graphnet/models/cnn/theos_muonE_upgoing.py` breaks with the snake-case convention. Do we strictly need "theos" in the module name? Proper credits can be given in the module docstring.